string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,4 @@
+/* exported isVowel */
+function isVowel(char) {
+  return (/^[aeiou]$/i.test(char));
+}
```

Comment:
I have a hard time believing you wrote a RegExp on your own. If you wrote this on your own, then nicely done, but if not, I urge you not to copy answers from the internet.

I would rewrite this on your own. A `switch` statement would be a great tool for this.